api-server/server/boot/settings.js

```diff
@@ -246,9 +246,7 @@ const updatePrivacyTerms = (req, res, next) => {
     }
     return res.status(200).json({
       type: 'success',
-      message:
-        'We have updated your preferences. ' +
-        'You can now continue using freeCodeCamp!'
+      message: 'We have updated your preferences. '
```

Comment:
```suggestion
      message: `We have updated your preferences.`
```

Drop the space and lets use backticks.